collect_app/src/test/java/org/odk/collect/android/activities/SplashScreenActivityTest.kt

```diff
@@ -67,56 +64,10 @@ class SplashScreenActivityTest {
         })
     }
 
-    @Test
-    fun `SplashScreenActivity should implement ProjectAddedListener`() {
```

Comment:
This test and the following further convinced me that the first launch screen should be its own activity.